tests/LargeXlsx.Tests/XlsxWriterTest.cs

```diff
@@ -447,4 +447,41 @@ public static void SharedStrings()
             sheet.Cells["A6"].Value.Should().Be("Lorem ipsum dolor sit amet");
         }
     }
+
+    [Test]
+    public static void HeaderFooter()
+    {
+        using var stream = new MemoryStream();
+        using (var xlsxWriter = new XlsxWriter(stream))
+        {
+            var headerFooter = 
+                new XlsxHeaderFooter(
+                    alignWithMargins: true,
+                    differentFirst: true,
+                    differentOddEven: false,
+                    scaleWithDoc: false,
+                    firstHeader: "&LFirstHeader",
+                    oddHeader: "&LLeftHeader&CCenterHeader&RRightHeader",
```

Comment:
I have not read what the standard says about the headerFooter section, but from this piece of code I understand there is some convention to specify up to three different texts for each header or footer. If that is the case, in my opinion EPPlus made this clearer, and we could try something similar.

Reply:
I agree EPPlus solution with constants is better approach than making developer to look up the standard documentation. I will try to resolve this.

Reply:
Added `XlsxHeaderFooterText` class what has properties for left, center and right string's. Adding the correct standard prefix code is handled during writing to worksheet based if the property is null or not.